index.html

```diff
@@ -0,0 +1,182 @@
+                        <h6 class="t-title">Workshop</h6>
+                        <p class="text-white p-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus ea sit
+                            error distinctio vitae.</p>
+                    </div>
+
+                    <div
+                        class="col-lg p-3 bg-cards d-flex flex-lg-column justify-content-center align-items-center gap-2">
+                        <img class="icons" src="src/img/app.png" alt="">
+                        <h6 class="t-title">CC Ignite</h6>
+                        <p class="text-white p-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus ea sit
+                            error distinctio vitae.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center mt-5 pb-5">
+                <a class="text-white text-center" href="#">SEE THE WHOLE PROGRAM</a>
+            </div>
+        </section>
+
+        <section class="container mt-5 d-flex flex-column">
+            <h2 class="text-center">Feature speakers</h2>
+            <div class="vector-container d-flex justify-content-center mb-4">
+                <div class="vector"></div>
+            </div>
+            <div class="cards-container row d-flex justify-content-center mt-5 mx-0">
+                
+            </div>
+            <button id="see_more" class="btn" type="button">SEE MORE<i class="fas fa-chevron-down"></i></button> 
+        </section>
+
+        <section class="companies py-5">
+            <h2 class="text-center text-secondary">Partner</h2>
+            <div class="vector-container d-flex justify-content-center mb-4">
+                <div class="vector"></div>
+            </div>
+            <ul class="d-lg-flex justify-content-center align-items-center gap-5">
+                <li class="li-companies text-center mt-4"><img src="src/img/mozilla.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/google.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/naver.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/daumkakao.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/airbnb.png" alt=""></li>
+            </ul>
+        </section>
+
+        <footer class="container gap-5 d-lg-flex my-2 justify-content-end">
+            <a class="navbar-brand text-secondary mx-5" href="#"><i class="fas fa-tram fa-3x">travel</i></a>
+            <div class="mt-3 mx-5">
+                <p class="fw-bold text-center">2022 Creative Commons Korea. Some Rigths Reserved. <span></span> </p>
+                <p class="text-center">Lorem ipsum dolor sit amet consectetur adipisicing elit. In quis quos ipsam,
+                    maxime voluptatum.</p>
+            </div>
+        </footer>
```

Comment:
The footer section in the mobile version should look almost identical to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design:
![image](https://user-images.githubusercontent.com/60292761/147277691-af9274b5-a66f-46f6-99f9-f0b75c01ea2e.png)

The original design : 
![image](https://user-images.githubusercontent.com/60292761/147277733-5a4cec04-2955-4201-b1f6-2b3971d5a497.png)

Please make sure to follow the original design as you can just make it horizontally.